plugins/com.google.cloud.tools.eclipse.preferences/schema/com.google.cloud.tools.eclipse.preferences.areas.exsd

```diff
@@ -0,0 +1,178 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!-- Schema file written by PDE -->
+<schema targetNamespace="com.google.cloud.tools.eclipse.preferences" xmlns="http://www.w3.org/2001/XMLSchema">
+<annotation>
+      <appinfo>
+         <meta.schema plugin="com.google.cloud.tools.eclipse.preferences" id="com.google.cloud.tools.eclipse.preferences.areas" name="Preference Areas"/>
+      </appinfo>
+      <documentation>
+         *** EXPERIMENTAL ***
+
+Provides a mini framework for hosting conceptually-related but independently-developed preferences areas on a single page.
+      </documentation>
+   </annotation>
+
+   <element name="extension">
+      <annotation>
+         <appinfo>
+            <meta.element />
```

Comment:
I don't think there is a "meta.element" element in the http://www.w3.org/2001/XMLSchema namespace